lib/livery/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Livery
-  VERSION = "0.1.0"
+  VERSION = "0.2.0"
```

Comment:
```suggestion
  VERSION = "0.2.1"
```
per my top level comment